api/src/model/play_asset_location.gd

```diff
@@ -0,0 +1,39 @@
+# ##############################################################################
+#
+#	Copyright 2020 Google LLC
+#
+#	Licensed under the Apache License, Version 2.0 (the "License");
+#	you may not use this file except in compliance with the License.
+#	You may obtain a copy of the License at
+#
+#		https://www.apache.org/licenses/LICENSE-2.0
+#
+#	Unless required by applicable law or agreed to in writing, software
+#	distributed under the License is distributed on an "AS IS" BASIS,
+#	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#	See the License for the specific language governing permissions and
+#	limitations under the License.
+#
+# ##############################################################################
+#
+# This class provides a constructor to create new PlayAssetLocation objects 
```

Comment:
suggestion:
`Wraps Play Core's AssetLocation which represents the location of an Asset within an asset pack on disk.`

See [here](https://github.com/google/play-unity-plugins/blob/master/GooglePlayPlugins/com.google.play.assetdelivery/Runtime/Scripts/AssetLocation.cs) for how the fields are described in the unity plugin.

Reply:
Updated the comments based on the suggestion. Also added comments for get methods based on PAD Java doc.